README.rst

```diff
@@ -16,7 +16,10 @@ View the published list of `Open edX Proposals (OEPs)`_ on ReadTheDocs.
 Testing locally
 ---------------
 
-To test locally in a Python virtual env::
+To test locally in a Python virtual env, you will first need to install `GraphViz <http://graphviz.org/>`_
+and either ``dot`` on the path or
+`adding options to the make call <https://groups.google.com/forum/#!topic/sphinx-users/yXgNey_0M3I>`_.
+On a Mac, this can be done via `brew install graphviz`. Next run the following commands::
```

Comment:
If you don't mind adding it: on Ubuntu, it can be installed with `sudo apt install graphviz`